src/core/styles/colors/utils.ts

```diff
@@ -0,0 +1,58 @@
+import { ColorClass, colorNames, Theme } from "./types";
+
+export const convertTailwindClassToVar = (className: string) =>
+  className.replace(/(text|bg|from|to)-([a-z0-9-]+)/gi, "var(--color-$2)");
+
+const calculatePaletteRange = (
+  acc: { max: number; min: number },
+  color: string,
+) => {
+  const splitColor = color.split("-");
+  const numericalColor = Number(splitColor[splitColor.length - 1]);
+  let { max, min } = acc;
+
+  if (min === -1 || numericalColor < min) {
+    min = numericalColor;
+  }
+
+  if (max === -1 || numericalColor > max) {
+    max = numericalColor;
+  }
+
+  return {
+    min,
```

Comment:
Can we do this here:
```

return {
    min: acc.min === -1 || numericalColor < acc.min ? numericalColor : acc.min,
    max: acc.max === -1 || numericalColor > acc.max ? numericalColor : acc.max,
  };
```

Reply:
That's a nice suggestion 👍 Cuts things down nicely